lib/elixir/pages/getting-started/modules-and-functions.md

```diff
@@ -7,7 +7,7 @@ iex> String.length("hello")
 5
 ```
 
-In order to create our own modules in Elixir, we use the `defmodule` macro. The first letter of the module must be in uppercase. We use the `def` macro to define functions in that module. The first letter of every function must be in lowercase (or underscore):
+In order to create our own modules in Elixir, we use the `defmodule/2` macro. The first letter of the module must be in uppercase. We use the `def/2` macro to define functions in that module. The first letter of every function must be in lowercase (or underscore):
```

Comment:
@josevalim lol great minds think alike... I just forgot to publish the review here 😩

Reply:
I initially had the same thought, but it seems the getting started guide is introducing it [from the start](https://hexdocs.pm/elixir/1.17.3/basic-types.html#identifying-functions-and-documentation) and using it elsewhere in this page as well as pages coming before (`case/2`, `if/2`...), so I wasn't sure.

Reply:
Had the same thought as @sabiwara. The getting started docs as far as I’ve seen consistently show the arity of similar keyword-like macros. Suggestion above would work though.

Reply:
Could also be that that's a remnant of us not being used to the exdoc trick above. I would rather change the instances above than use arity here.

Reply:
SGTM, but in this case we'd need to rework the guides to introduce the concept of arity later and go through all instances, so it's not a trivial change.

Reply:
@sabiwara I disagree that we need to introduce arity later. We can introduce arity where we do now, but keep the prose and docs simpler here and just use `defmodule` and co rather than the arity variant.

Put it another way: what does the arity add here for the user?

Reply:
> We can introduce arity where we do now, but keep the prose and docs simpler here and just use defmodule and co rather than the arity variant.

My only concern if the notation starts appearing far after the explanation, it could be puzzling to the reader as they might not remember?

> Put it another way: what does the arity add here for the user?

Arguably, familiarity with how to read Elixir docs, which at some point will be desirable, but I agree could be overwhelming at the very beginning.
Which is why I thought it might be best to not use it before introducing arity, and to start using it from this point on, giving concrete examples of what they just learned?

But I don't feel strongly about this, your proposal sounds good to me too.